CODEOWNERS

```diff
@@ -0,0 +1,70 @@
+# This is a comment.
+# Each line in this file is a file pattern followed by one or more
+# @github_usernames and/or e-mail addresses corresponding to users with write
+# permissions for the repository.
+# https://help.github.com/en/articles/about-code-owners
+
+# Order is important; the user(s) in the last matching pattern here for a file will
+# be that file's owner(s).
+
+# The users below will be the default owners for everything in the repository.
+# Unless a line further in this file matches, all of the users here will be
+# requested for review when someone opens a pull request.
+*	dave@picknik.ai me@v4hn.de mferguson@fetchrobotics.com moveit_releasers@googlegroups.com
+
+# The lines below correspond to directories with a package.xml file and the
+# maintainers specified in that file
+
+/moveit_runtime/	gm130s@gmail.com
+
+/moveit_plugins/moveit_plugins/	me@v4hn.de moveit_releasers@googlegroups.com
```

Comment:
remove, this isn't a real package (its empty)

Reply:
Done